src/main/webapp/login-status.js

```diff
@@ -15,8 +21,13 @@ function signOut() {
 
 // Get whether user is signed in
 function getLoginStatus() {
-  const auth2 = gapi.auth2.getAuthInstance();
-  return auth2.isSignedIn.get();
+  const auth2 = gapi.auth2;
```

Comment:
Noticed while working on history UI this should be a let since it's reassigned in else block

Reply:
nice catch! fixed.